app/code/Magento/GiftMessage/view/frontend/requirejs-config.js

```diff
@@ -7,9 +7,7 @@ var config = {
     map: {
         '*': {
             giftOptions:    'Magento_GiftMessage/js/gift-options',
-            extraOptions:   'Magento_GiftMessage/js/extra-options',
-            'Magento_GiftMessage/gift-options':    'Magento_GiftMessage/js/gift-options',
-            'Magento_GiftMessage/extra-options':   'Magento_GiftMessage/js/extra-options'
+            'Magento_GiftMessage/gift-options':    'Magento_GiftMessage/js/gift-options'
```

Comment:
Hello @eduard13 
Perhaps this is redundant mapping
Could you please check it?
Thank you in advance.

Reply:
I agree that is a redundant declaration, however the purpose of this task/PR is another - to get rid of deprecated code.

Reply:
Ok. Thanks.
Approved